packages/react-scripts/scripts/openPlayer.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+const chalk = require('react-dev-utils/chalk');
+const fs = require('fs');
+const { execFile } = require('child_process');
+
+async function openPlayer(url) {
+    return new Promise((_, reject) => {
+        if (!process.env.PLAYER_PATH) {
+            console.error(chalk.red('PLAYER_PATH environment variable not set. Set it in the ".env" file to run the player.'));
+            reject();
+        }
+
+        if (path.extname(process.env.PLAYER_PATH) !== '.exe') {
+            console.error(chalk.red('PLAYER_PATH environment variable should point to the player executable. Set it in the ".env" file to run the player.'));
+            reject();
+        }
+
+        const playerFile = path.extname(process.env.PLAYER_PATH) ? '' : './Player.exe';
+
+        const playerPath = path.resolve(__dirname, process.env.PLAYER_PATH, playerFile);
```

Comment:
Why are we doing that? In the previous check, we are making sure the player path is executable. So isn't it enough to directly use the Player_Path variable?
Or we are making sure its name is Player.exe? I am a bit confused about these lines :D

Reply:
I did it because I was testing something first. When I started I did a check if the provided file exists so I did a check only for that. However I then realized, that `fs.exists` checks a path is correct, so if I pass a path and not the Player it will still return that it's correct. So I did this - however it's obsolete when I'm checking the extensions, since paths don't have extensions 😅